js/cards.js

```diff
@@ -186,9 +186,9 @@ function setupHover() {
   for (let i = 0; i < cardList.length; i++) {
     cardList[i].addEventListener("animationend", removeAnimation);
     cardList[i].parentElement.addEventListener("blur", removeAnimation);
-    cardList[i].addEventListener("mouseover", addAnimation);
     cardList[i].parentElement.addEventListener("focus", addAnimation);
-    cardList[i].addEventListener("click", addAnimation);
+    cardList[i].parentElement.addEventListener("click", addAnimation);
+    cardList[i].parentElement.addEventListener("mouseover", addAnimation);
```

Comment:
Not sure if this change is what caused it since it looks like there was only rearranging here, but the cards below the topmost one are reanimating on hover

This doesn't seem to be happening on the main branch (at least on hover, I seem to be able to click into cards below)

Reply:
I put the order back to where it was, but it's also now targeting the parent (button) element instead of the card-image div. Was trying to finagle which element to capture events so that a user could scroll on mobile while touching the cards.